lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
OpenBSD:

```
~$ getconf _NPROCESSORS_ONLN
getconf: _NPROCESSORS_ONLN: unknown variable
~$ getconf NPROCESSORS_ONLN
1
```

But of course on Linux:

```
~% getconf NPROCESSORS_ONLN   
getconf: Unrecognized variable `NPROCESSORS_ONLN'
~% getconf _NPROCESSORS_ONLN
4
```

sigh.

How about an ENV var that can override this?

```ruby
def number_of_cores
  ENV.fetch("NUMBER_OF_CORES", %{`getconf _NPROCESSORS_ONLN`})
end
```